index.js

```diff
@@ -8,7 +8,9 @@ const kWs = Symbol('ws-socket')
 const kWsHead = Symbol('ws-head')
 
 function fastifyWebsocket (fastify, opts, next) {
-  let errorHandler = defaultErrorHandler
+  fastify.decorateRequest('ws', null) // this is a decorator for the Request ws property
```

Comment:
I think these comments are a little overly verbose, we try to only comment stuff that is not obvious or has hidden semantics that you can't figure out by just reading the code. In this case I think what the code does is obvious such that the comment really just repeats the same information, you know?

Reply:
Sorry, I don't comment my code normally.